source/antwerp/SLURM_PBS_differences.rst

```diff
@@ -10,14 +10,14 @@ Important differences between Slurm and Torque
      This can have unexpected results, e.g., if you resubmit the job from a different
      environment or if some things are in different directories on the login and cluster
      nodes which does sometimes happen when we do a silent upgrade of the cluster.
-       - ``--get-user-env`` will give an environment pretty equivalent
-         to what you would get on Torque
-       - ``--export=NONE`` will start the job with a very empty environment
+      - ``--get-user-env`` will give an environment pretty equivalent
+        to what you would get on Torque
```

Comment:
in what sense does this give a torque like environment? It still pass the environment you're in to the job?